README.rst

```diff
@@ -2,8 +2,8 @@
 Exposure Log
 ############
 
-Exposure log is a web service to create and manage log messages associated with a particular exposure.
-It is self-documented using OpenAPI.
+Exposure log is a REST web service to create and manage log messages that are associated with a particular exposure.
+It is self-documenting using OpenAPI.
```

Comment:
How would a user know to look for the docs unless this information is in README?

Reply:
I recommended removing it because it's a superfluous fact about the app that doesn't add to the description of what it does (you could also write that it's built with FastAPI and Python and deployed with Kubernetes... but none of those facts help to describe the app as well as the first sentence did. 

The very next paragraph of the README provides a path for the HTTP API documentation, and that's really you need to get the point across.

Reply:
Good point. Fixed.